kvdb-rocksdb/src/lib.rs

```diff
@@ -642,21 +659,17 @@ impl Database {
 
 	/// The number of non-default column families.
 	pub fn num_columns(&self) -> u32 {
-		self.db
-			.read()
-			.as_ref()
-			.and_then(|db| if db.cfs.is_empty() { None } else { Some(db.cfs.len()) })
-			.map(|n| n as u32)
-			.unwrap_or(0)
+		self.config.memory_budget.len() as u32
 	}
 
 	/// Drop a column family.
-	pub fn drop_column(&self) -> io::Result<()> {
+	pub fn drop_column(&mut self) -> io::Result<()> {
```

Comment:
So we can only drop columns in last-first order? That's an odd API isn't it?

Reply:
yes, it is, but I'm not even sure if we've ever used it

Reply:
Can we get rid of it?

Reply:
I mean technically we use it here https://github.com/paritytech/parity-ethereum/blob/db1ea1dcd8361c6b53a136842398464da363447b/util/migration-rocksdb/src/lib.rs#L304, but in practice, we never dropped columns on db version bump.